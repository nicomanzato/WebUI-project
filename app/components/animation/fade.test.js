import React from 'react'
import Fade from './fade'
import {
  StyleSheet,
  View,
  Text,
  Image,
  TouchableHighlight
} from 'react-native';
import { shallow, mount, ReactWrapper } from 'enzyme'

describe('fade animation', () => {

  it('should render correctly', () => {
    const wrapper = shallow(
      <Fade visible={true}>
        <Text> This is a test </Text>
      </Fade>
    );

    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly', () => {

    const wrapper = shallow(
      <Fade visible={false}>
        <Text> This is a test </Text>
      </Fade>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
