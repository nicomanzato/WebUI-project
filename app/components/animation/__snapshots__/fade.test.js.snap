// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fade animation should render correctly 1`] = `
<AnimatedComponent
  style={
    Array [
      Object {
        "opacity": 0,
        "transform": Array [
          Object {
            "scale": 1.1,
          },
        ],
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
     This is a test 
  </Text>
</AnimatedComponent>
`;

exports[`fade animation should render correctly 2`] = `
<AnimatedComponent
  style={
    Array [
      Object {
        "opacity": 0,
        "transform": Array [
          Object {
            "scale": 1.1,
          },
        ],
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
     This is a test 
  </Text>
</AnimatedComponent>
`;
